mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreRightManager.java

```diff
@@ -104,13 +105,9 @@ public Rfc4314Rights myRights(MailboxId mailboxId, MailboxSession session) throw
 
     @Override
     public Mono<Rfc4314Rights> myRightsReactive(MailboxId mailboxId, MailboxSession session) {
-        try {
-            MailboxMapper mapper = mailboxSessionMapperFactory.getMailboxMapper(session);
-            return mapper.findMailboxByIdReactive(mailboxId)
-                .map(Throwing.function(mailbox -> myRights(mailbox, session)));
-        } catch (MailboxException e) {
-            return Mono.error(e);
-        }
+        MailboxMapper mapper = mailboxSessionMapperFactory.getMailboxMapper(session);
```

Comment:
Could be for fully reactive?
```
return Mono.fromCallable(() -> mailboxSessionMapperFactory.getMailboxMapper(session))
.flatMap(mapper -> mapper.findMailboxByIdReactive(mailboxId).map(Throwing.function(mailbox -> myRights(mailbox, session)));
```

Reply:
mailboxSessionMapperFactory.getMailboxMapper(session) do not imply any long or costly calls, thus I think it is unecessary to wrap it with reactor